articles/active-directory-b2c/openid-connect.md

```diff
@@ -135,59 +135,63 @@ After the ID token is validated, you can begin a session with the user. You can
 
 ## Get a token
 
-If you need your web application to only run user flows, you can skip the next few sections. These sections are applicable only to web applications that need to make authenticated calls to a web API and are also protected by Azure AD B2C.
+If you need your web application to only run user flows, you can skip the next few sections. These sections are applicable only to web applications that need to make authenticated calls to a web API which is also protected by Azure AD B2C.
 
-You can redeem the authorization code that you acquired (by using `response_type=code+id_token`) for a token to the desired resource by sending a `POST` request to the `/token` endpoint. In Azure AD B2C, you can [request access tokens for other APIs](access-tokens.md#request-a-token) as usual by specifying their scope(s) in the request.
+You can redeem the authorization code that you acquired (by using `response_type=code`) for a token to the desired resource by sending a `POST` request to the `/token` endpoint. In Azure AD B2C, you can [request access tokens for other APIs](access-tokens.md#request-a-token) as usual by specifying their scope(s) in the request.
 
-You can also request an access token for your app's own back-end Web API by convention of using the app's client ID as the requested scope (which will result in an access token with that client ID as the "audience"):
+You can also request an access token for your app's own back-end Web API by convention of using the app's **Application ID URI** plus **Scope** name (`{application-id-uri}/{scope-name}`) as the requested scope (which will result in an access token with the desired resource's client ID as the "audience"):
```

Comment:
```suggestion
You can also request an access token for your app's own back-end Web API by convention of using the app's client ID as the requested scope (which will result in an access token with that client ID as the "audience"):
```

Reply:
Do not make any change here! You use app ID (no need of scope) when an app wants to access an api of itself.

Reply:
Agreed!
I got to caught up in the first leg of the authorization_code flow to even think that this section mentions when the application is calling itself as a backend API (and regardless if that is best practice or not, it's certainly true that it works) 👍 
Thanks for saving me 🥇  from publishing incorrect content, when I tested I was another app registration as API...